CMakeLists.txt

```diff
@@ -244,24 +269,47 @@ configure_file(liborcania.pc.in liborcania.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/liborcania.pc
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 
-set(TARGETS orcania)
+set(TARGETS )
+if (BUILD_SHARED)
+    list(APPEND TARGETS orcania)
+endif ()
 if (BUILD_STATIC)
-    set(TARGETS ${TARGETS} orcania_static)
+    list(APPEND TARGETS orcania_static)
 endif ()
 
 if (INSTALL_HEADER)
-    install(TARGETS ${TARGETS}
+    install(TARGETS ${TARGETS} EXPORT OrcaniaExports
             RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
             LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
             ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
             PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 else ()
-    install(TARGETS ${TARGETS}
+    install(TARGETS ${TARGETS} EXPORT OrcaniaExports
             RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
             LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
             ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif ()
 
+set(ORCANIA_INSTALL_CMAKEDIR_DEFAULT "${CMAKE_INSTALL_LIBDIR}/cmake/Orcania")
+if (WIN32 AND NOT MINGW)
+    set(ORCANIA_INSTALL_CMAKEDIR_DEFAULT "cmake")
+endif ()
+set(ORCANIA_INSTALL_CMAKEDIR ${ORCANIA_INSTALL_CMAKEDIR_DEFAULT} CACHE STRING "Location where to install the cmake config files")
+
+install(EXPORT OrcaniaExports DESTINATION "${ORCANIA_INSTALL_CMAKEDIR}"
+        NAMESPACE "Orcania::"
+        FILE "OrcaniaTargets.cmake")
+
+configure_package_config_file(cmake-modules/OrcaniaConfig.cmake.in OrcaniaConfig.cmake
+        INSTALL_DESTINATION "${ORCANIA_INSTALL_CMAKEDIR}")
+write_basic_package_version_file(OrcaniaConfigVersion.cmake
+        COMPATIBILITY AnyNewerVersion)
+
+install(FILES
```

Comment:
IMHO, the `/usr/lib/cmake/Orcania/` files should be installed only in development mode, i.e. when the option `INSTALL_HEADER` is `ON`, don't you think?

Reply:
Sounds good. Will do that.

Reply:
Fixed in 52735b0